components/rmrk/Gallery/History.vue

```diff
@@ -339,26 +336,13 @@ export default class History extends mixins(
           break
         case InteractionBsxOnly.ROYALTY:
           event['From'] = newEvent['caller']
-          if (!this.query) {
-            this.query = await resolveQueryPath(this.urlPrefix, 'nftById')
-          }
-          if (this.query && nftId !== 'id') {
-            const nft = await this.$apollo.query({
-              query: this.query.default,
-              client: this.urlPrefix,
-              variables: {
-                id: nftId,
-              },
-            })
-            const {
-              data: { nftEntity },
-            } = nft
-            const amount =
-              nftEntity.price && nftEntity.royalty
-                ? royaltyOf(nftEntity.price, nftEntity.royalty)
-                : ''
-            event['Amount'] = this.parsePrice(amount)
-          }
+          event['To'] = ''
+          event['Percentage'] = parseInt(newEvent['meta'])
```

Comment:
`toPercent from @/utils/filters`

Reply:
Already use `Vue filter` `toPercent` in
```
<b-table-column
            cell-class="short-identity__table"
            :visible="isPercentageColumnVisible"
            field="Percentage"
            label="Percentage"
            v-slot="props">
            <span :class="percentageTextClassName(props.row.Percentage)">
              {{ props.row.Percentage | toPercent('-') }}
            </span>
</b-table-column>
```

So I only need to give a number to `event['Percentage']`